.github/workflows/update-caches.yml

```
@@ -0,0 +1,136 @@
+name: Update Actions Caches
+
+on:
+  # Manually
+  workflow_dispatch:
+  # On PR merge
+  push:
+    branches:
+      - main
+  # After nightly release
+  schedule:
+    - cron: "0 1 * * *"
+
+env:
+  CARGO_INCREMENTAL: 0
+  CARGO_PROFILE_TEST_DEBUG: 0
+  CARGO_PROFILE_DEV_DEBUG: 0
```

```suggestion
  CARGO_PROFILE_TEST_DEBUG: line-tables-only
  CARGO_PROFILE_DEV_DEBUG: line-tables-only
```

(Nit) Completely removing debug info means that backtraces will not have file names and line numbers, which can make it harder to diagnose bugs. `line-tables-only` includes these things, while still majorly shrinking artifact size.

---

FYI these need to match the env variables used in `ci.yml` (otherwise rustc won't reuse the build artifacts and the cache will be useless), and those use `0`.

Might be another thing worth documenting to avoid someone modifying the env vars in one place but not in the other.